sklearn/base.py

```diff
@@ -411,6 +435,9 @@ def fit_predict(self, X, y=None):
         self.fit(X)
         return self.labels_
 
+    def _get_tags(self):
+        return _update_tags(self, super(ClusterMixin, self), is_clusterer=True)
```

Comment:
should the reason for the apparently weird super() here be written down somewhere?